RFC/src/Glossary.md

```diff
@@ -126,10 +130,10 @@ A Communication Node is either a Validator Node or Base Node that is part of the
 A Communication Client is a Wallet or Asset Manager that makes use of the Tari communication network to send joining and discovery requests. A Communication Client does not maintain the communication network and is not responsible for forwarding or propagating any requests or data messages.
 
 
-## Creator Assigned Mode
-[creator assigned mode]: #creator-assigned-mode
+## Creator Nomination Mode
```

Comment:
perhaps add here that there is no public broadcast invitation

Reply:
I feel the word direction of the nomination and the word `directly` does convey that idea.

Reply:
It does kinda, but it does not implicitly state that, which I feel is better in a RFC.